eahub/base/views.py

```diff
@@ -130,6 +140,48 @@ class RobotsTxtView(base.TemplateView):
     template_name = "robots.txt"
     content_type = "text/plain; charset=utf-8"
 
+class ReportAbuse:
```

Comment:
There doesn't seem to be any strong reason for this to be a separate class from the view.